Extension/src/LanguageServer/configurations.ts

```diff
@@ -741,6 +742,18 @@ export class CppProperties {
         return util.resolveVariables(property, env);
     }
 
+    private updateConfigurationBoolean(property: boolean | undefined | null, defaultValue: boolean | undefined | null): boolean | undefined {
+        if (property === null || property === undefined) {
+            property = defaultValue;
+        }
+
+        if (property === null) {
+            return property = undefined;
```

Comment:
Is this supposed to return `undefined` instead of `property = undefined`?

Reply:
Yep, silly mistake, thanks.